apps/docs/src/app/platform/component-docs/platform-action-bar/platform-action-bar-examples/platform-action-bar-positive-and-negative-action-example.component.html

```diff
@@ -1,5 +1,6 @@
 <fdp-action-bar
     id="actionbar_with_menu"
+    [contentDensity]="'compact'"

```

Comment:
you have everywhere contentDensity=compact, you might want to have some examples with cozi, or have some dropdown to pick

Reply:
added example.